### PR TITLE
add frature bulk create library

### PR DESCRIPTION
**Pull request by Harshtroo**

### CODE REVIEW

**Comment by Harshtroo** [comment]
Bulk create

File: book/book_crud/static/js/create_library.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ $("#my-create-library").on("submit",function(event){
     event.preventDefault()
     var csrfToken = $('input[name="csrfmiddlewaretoken"]').val();
     var form_data = new FormData($("#my-create-library")[0]);
+    console.log("form_data=========",form_data)
     var Callback = function(response) {
```
Comment (Dhruvi-Makwana) [inline book/book_crud/static/js/create_library.js]: Remove console

File: book/book_crud/views.py [hunk 2]
```diff
@@ -111,14 +112,24 @@ def user_list(request):
...
+    serializer_class = LibrarySerializer
+
+    def get_serializer(self,*args,**kwargs):
+        if isinstance(kwargs.get("data",{}),list):
+            kwargs["many"] = True
+        return super(LibraryView,self).get_serializer(*args,**kwargs)
+
 
 @api_view(["GET"])
 def library_list(request):
```
Comment (Dhruvi-Makwana) [inline book/book_crud/views.py:149]: if possible create classbased view

File: book/book_crud/views.py [hunk 3]
```diff
@@ -111,14 +112,24 @@ def user_list(request):
...
+
+
+class LibraryView(generics.CreateAPIView):
+    serializer_class = LibrarySerializer
+
+    def get_serializer(self,*args,**kwargs):
+        if isinstance(kwargs.get("data",{}),list):
+            kwargs["many"] = True
+        return super(LibraryView,self).get_serializer(*args,**kwargs)
+
```
Comment (Dhruvi-Makwana) [inline book/book_crud/views.py:146]: class RegisterView(CreateAPIView):
    permission_classes = [permissions.IsAuthenticated]
    serializer_class = UserSerializer
    you can also try this.

Comment (Harshtroo) [reply book/book_crud/views.py:146]: not a register view this is a library create view

**Pull request merged**